README.md

```diff
@@ -36,6 +36,8 @@ GLOBAL OPTIONS:
    --dry-run, -d                       Conduct a dry-run: Repackage the image, but only write the Lambda layers to local disk (do not publish to Lambda)
    --description value, --desc value   The description of this layer version
    --license-info value, -l value      The layer's software license. It can be an SPDX license identifier, the URL of the license hosted on the internet, or the full text of the license
+   --compatible-runtime value --compatible-runtime value,
+                --cr value --cr value  A list of compatible function runtimes with this layer. (default: "provided")
```

Comment:
This needs to be updated, below has "An AWS Lambda function runtime compatible with the image layers..."

Reply:
done